src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -291,8 +284,6 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
         HttpHeaders responseHeaders;
         programmingExerciseImportService.importRepositories(originalProgrammingExercise, importedProgrammingExercise);
         try {
-            // TODO: We have removed the automatic build trigger from test to base for new programming exercises. We need to also remove this build trigger manually on the case of
-            // an import as the source exercise might still have this trigger.
```

Comment:
Is the todo really resolved? If yes, we can keep the comment as documentation, right?

If not, we should keep it.

Reply:
Yes, this was resolved about two months ago, when we merged the triggerUpdate parameters for build plan updates.
I removed the TODO and updated the comment to fit the current implementation